src/routes/docs/references/[version]/models/[model]/+page.server.ts

```diff
@@ -22,29 +31,24 @@ export const load: PageServerLoad = async ({ params }) => {
         title: schema.description as string,
         properties: props.map(([name, data]) => {
             const property = data as AppwriteSchemaObject;
+            let learnMore = ' Learn more about the: ';
```

Comment:
lets try avoid having text represented here invariables , lets rather have the url/urls passed from the load function and do the rest on the UI side.

Reply:
Done